server/debian.Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM debian:10-slim
+
+LABEL varion="2.0" \
+    maintainer="ESh4d0w, Markus Hadenfeldt, docker@teaspeak.de, h1dden-da3m0n" \
+    description="A simple TeaSpeak server running on debian 10 (amd64)"
+
+ARG TEASPEAK_VERSION="1.4.12"
+
+RUN	apt-get update && apt-get upgrade -qqy \
+    && apt-get install --no-install-recommends -qqy \
+         ca-certificates wget curl python libnice10 ffmpeg tzdata \
+    \
+    && mkdir ts && cd /ts \
+    && wget -nv -O TeaSpeak.tar.gz https://repo.teaspeak.de/server/linux/amd64/TeaSpeak-$TEASPEAK_VERSION.tar.gz \
```

Comment:
Used `amd64_<channel>` here. `amd64` only is outdated and only for legacy reasons there.
I recommend using the `amd64_stable` channel (currently `amd64` is linked to that btw)

Reply:
oh, okay, simply used the old url from your Dockerfile without checking if there was any other way to get to the file 😅